content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md

```diff
@@ -1,84 +1,46 @@
-                <div class="story-header__label">
-                  <h2>Paul Evason, <br/>Assistant headteacher</h2>
-                </div>
-               </div>
-         <p class="prominent">
-          I always wanted to work in an environment that was never going to be the same every day – teaching definitely provides this. 
-          Working with students from different backgrounds and preparing them to take their next steps towards success was 
-          always a huge draw to the profession.
-          </p>
-          <p>As a teacher, you’ll have the opportunity to develop and succeed in your career to realise your professional and personal ambitions.
-          Before I was assistant headteacher I enjoyed a number of different roles learning new skills 
-       that have helped me to move forward in my career.
-         </p>
-         <div>
-           <div class="quote-block">
-               <span class="icon-quote"></span>
-                   <strong class="quote-block__content">There are plenty of progression opportunities in teaching – you just have to grasp them.
-                   <span class="icon-quote quote-close"></span></strong>
-             </div>
-        <h3>Take on a challenge</h3>
-              <p>When I started teaching as a newly qualified teacher, I looked for opportunities to support the school 
-              and to enhance my experience.</p> 
-              <p>For example, in my first year, I led on re-designing the KS3 curriculum, and in my second year I started to 
-         become involved in the school’s virtual learning environment. 
-         All of this added to my experience and provided me with plenty to talk about when applying for new roles.
-	        </p>
-          </div>
-          <p>My long-term plan is to go into headship, but do not want to rush this. I want to be able to experience a range of different areas in
-         schools to ensure that I am fully prepared for the challenges that a headship role will bring.
-         </p>
-         <p>I feel fortunate to be on the leadership salary, but I don’t do the job for the money – for me it’s about supporting students as a
-         teacher and leader and making sure they achieve their full potential.
-         </p>
-         </div>
-    </div>
-</div>
+I always wanted to work in an environment that was never going to be the same every day – teaching definitely provides this. Working with students from different backgrounds and preparing them to take their next steps towards success was always a huge draw to the profession.
+
+As a teacher, you'll have the opportunity to develop and succeed in your career to realise your professional and personal ambitions. Before I was assistant headteacher I enjoyed a number of different roles learning new skills that have helped me to move forward in my career.
+
+> There are plenty of progression opportunities in teaching – you just have to grasp them.
+
+###Take on a challenge
```

Comment:
Can't remember if Kramdown allows it but its a bit easier to read with a space in it `### Take on a challenger`

Reply:
Agree, will revisit and tidy.